### DISCUSSION TITLE
Template for new note creation?

### DISCUSSION THREAD

**Discussion by jli** [svsool/vscode-memo#600] [Q&A]
I think it'd be a great workflow if memo supported a template/snippet when creating a new note (triggered when following a link to a new note that doesn't exist, or maybe via a Code Action thing).

The basic behavior I'd like is just inserting `# <note link text>`, though I can imagine making it a full-on template or otherwise more configurable would be a nice general mechanism. (One thing that'd be especially neat: auto-convering "snake_case_link_text" to "Title Case Link Text".)

Is there any way to configure this, or other workarounds I could use?

Thanks!

**Comment by svsool** [comment]
Hi @jli, similar question was already answered in [this issue](https://github.com/svsool/memo/issues/572#issuecomment-1200111588).
